### PR TITLE
Add example of usage with @storeon/router

### PR DESCRIPTION
**Pull request by npupko**
As discussed in https://github.com/storeon/router/issues/13

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -96,3 +96,32 @@ Import `connect` function from our `./store` and use it for getting state and di
...
+  router.createRouter([
+    ['/', () => ({ page: 'home' })],
+    ['/blog', () => ({ page: 'blog' })],
+  ])
+])
+```
+
+And use it like:
+#### `App.svelte`
+```svelte
```
Comment (distolma) [inline README.md:118]: does github recognize `svelte` as syntax highlight? maybe better to use `html`?

Comment (npupko) [reply README.md:118]: @distolma Yeah, github recognizes it pretty well. You can take a look at [the blob of this file](https://github.com/storeon/svelte/blob/d9c931683ac404bf9fdd09f3c252ab9891a1316b/README.md)

File: README.md [hunk 2]
```diff
@@ -96,3 +96,32 @@ Import `connect` function from our `./store` and use it for getting state and di
...
+
+## Usage with [@storeon/router](https://github.com/storeon/router)
+If you want to use the @storeon/svelte with the [@storeon/router](https://github.com/storeon/router) you should import the `router.createRouter` from [@storeon/router](https://github.com/storeon/router) and add this module to `createSvelteStore` instead of `createStore`
+
+#### `store.js`
+```js
+import { createSvelteStore } from "@storeon/svelte";
+import router from '@storeon/router'
+const connect = createSvelteStore([
+  router.createRouter([
```
Comment (distolma) [inline README.md]: it's may not matter, but better to follow one style guide, for example
```js
import { createSvelteStore } from "@storeon/svelte";
import { createRouter } from "@storeon/router";

const connect = createSvelteStore([
createRouter([
...
```

File: README.md (hunk 1 above)
Comment (npupko) [reply README.md:118]: @distolma It correctly add `highlight highlight-source-svelte` classes to the div with this code:)

File: README.md (hunk 2 above)
Comment (npupko) [reply README.md]: Done

File: README.md [hunk 3]
```diff
@@ -96,3 +96,33 @@ Import `connect` function from our `./store` and use it for getting state and di
...
+  ])
+])
+```
+
+And use it like:
+#### `App.svelte`
+```svelte
+<script>
+  import { connect } from "./store.js";
+  import router from '@storeon/router'
```
Comment (distolma) [inline README.md]: use double quotes for string

Comment (npupko) [reply README.md]: Done

**Pull request merged**